vignettes/word_cloud.Rmd

```diff
@@ -0,0 +1,80 @@
+---
+title: Word cloud of swear-words from reddit comments
+author: Related to Trump
+output:
+  html_document:
+    mathjax:  default
+    fig_caption:  true
+    toc: true
+    section_numbering: true
+    css: ggsci.css
+vignette: >
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteIndexEntry{Word cloud of swear-words from reddit comments}
+---
+```{r include = FALSE}
+library(knitr)
+opts_chunk$set(
+	comment = "",
+	fig.width = 12, 
+	message = FALSE,
+	warning = FALSE,
+	tidy.opts = list(
+		keep.blank.line = TRUE,
+		width.cutoff = 150
+		),
+	options(width = 150),
+	eval = TRUE
+)
+```
+
+
+```{r}
+library(sweary);library(dplyr)
+swear_words %>%
+  count(language)
+```
+
+
+
+```{r}
+library(RedditExtractoR)
+reddit_comments <-
```

Comment:
This piece of code produces a lot of mess while downloading the comments.

|                                                                                                                                                  
  |                                                                                                                                            |   0%
  |                                                                                                                                                  
  |======                                                                                                                                      |   4%
  |                                                                                                                                                  
  |===========                                                                                                                                 |   8%

We don't want that in the vignette.